content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

```diff
@@ -0,0 +1,263 @@
+specific locations in the warehouse as loading zones for delivery trucks.
+
+Then, once all the configurations are complete, be sure to click :guilabel:`Save`.
+
+.. _inventory/shipping_receiving/delivery-method-for-carrier:
+
+Delivery method
+---------------
+
+Next, assign each delivery vehicle as a *Carrier* by configuring a delivery method.
+
+To configure delivery methods, go to :menuselection:`Inventory app --> Configuration --> Delivery
+Methods`. Select an existing delivery method, or click :guilabel:`New`.
+
+.. seealso::
+   :doc:`Configure delivery method <../setup_configuration>`
+
+On the :guilabel:`Delivery Method` form, enter a name for the delivery method. It is recommended to
+use identifying information, such as the vehicle description and license plate number (e.g. `Truck
+123-ABCD`).
+
+Since the delivery methods are managed internally, set the :guilabel:`Provider` to either
+:guilabel:`Fixed Price` or :guilabel:`Based on Rules`. For more information about how shipping
+prices are calculated, refer to the :doc:`Delivery method <../setup_configuration>` article.
+
+Next, set a :guilabel:`Delivery Product`, which is the product that shows up as the customer's
+:ref:`delivery charge <inventory/shipping_receiving/delivery-product>` on the sales order or
+invoice.
+
+Optionally, in the :guilabel:`Availability` tab, set the :guilabel:`Countries`, :guilabel:`States`,
+or :guilabel:`Zip Prefixes` to limit the range of local delivery.
+
+.. figure:: dispatch/delivery-method.png
+   :alt: Delivery method form.
+
+   Example delivery method, with the :guilabel:`Zip Prefixes` set to San Francisco's zip code.
+
+.. _inventory/shipping_receiving/docks:
+
+Dock locations
+--------------
+
+Each loading dock must have a dedicated location. To create or configure dock locations, go to
+:menuselection:`Inventory app --> Configuration --> Locations`.
+
+Click the desired location, which opens the :guilabel:`Location` form. In the :guilabel:`Additional
+Information` section, tick the :guilabel:`Is a Dock Location` checkbox.
+
+.. figure:: dispatch/dock-location.png
+  :alt: Location configuration.
+
+  Location configuration page with :guilabel:`Is a Dock Location` checkbox ticked.
+
+Build loads
+===========
+
+Once setup is complete, :ref:`assign orders to a carrier
+<inventory/shipping_receiving/assign-carrier>` and :ref:`group them into batches
+<inventory/shipping_receiving/create-batch>`. Then, :ref:`configure the batch form
+<inventory/shipping_receiving/batch-form>`, as needed.
+
+Since this article is about a specific use case, explore details about each picking method in their
+dedicated articles.
+
+.. seealso::
+   - :doc:`../picking_methods/batch`
+   - :doc:`Wave picking <../picking_methods/wave>`
+   - :doc:`../picking_methods/cluster`
+
+To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which
+reveals a list of outgoing deliveries.
```

Comment:
Another suggestion to swap seealso position and text, just to avoid breaking the flow of the paragraph
```suggestion
To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which
reveals a list of outgoing deliveries.

.. seealso::
   Since this article is about a specific use case, explore details about each picking method in
   their dedicated articles.

   - :doc:`../picking_methods/batch`
   - :doc:`Wave picking <../picking_methods/wave>`
   - :doc:`../picking_methods/cluster`

```